alacritty/Cargo.toml

```diff
@@ -34,6 +34,7 @@ libc = "0.2"
 unicode-width = "0.1"
 bitflags = "1"
 dirs = "3.0.1"
+signal-hook = "0.3.9"
```

Comment:
Seems like there have been a lot of updates to `signal-hook`, we should update the version in `alacritty_terminal/Cargo.toml` to keep things consistent.

Reply:
Done.